rip/core.py

```diff
@@ -878,21 +879,43 @@ def prompt_creds(self, source: str):
         :type source: str
         """
         if source == "qobuz":
-            secho("Enter Qobuz email:", fg="green")
-            self.config.file[source]["email"] = input()
-            secho(
-                "Enter Qobuz password (will not show on screen):",
-                fg="green",
-            )
-            self.config.file[source]["password"] = md5(
-                getpass(prompt="").encode("utf-8")
-            ).hexdigest()
+            secho("Use Qobuz auth token to authenticate? (yes/no)", fg="green")
+            use_auth_token = input()
 
-            self.config.save()
-            secho(
-                f'Credentials saved to config file at "{self.config._path}"',
-                fg="green",
-            )
+            answer_is_true = re.match("(?i)^y", use_auth_token) is not None
+            if USE_AUTH_TOKEN_DEFAULT is False:
```

Comment:
You should remove this constant entirely. The default value is stored in the config.toml file already.

Reply:
Added this suggestion